tests/test_pipeline.py

```diff
@@ -115,13 +115,13 @@ def test_interval_constraints(type_: Any, pipeline: Any, valid_cases: list[Any],
     [
         (
             str,
-            validate_as(str).len(min_len=2, max_len=5),
+            validate_as(str).len(min_len=2, max_len=5),  # type: ignore
```

Comment:
Same q, why the ignore?

Reply:
This one was because some of the types like `list` where missing parametrization below (needs to be `list[int]`)